File: src/Learn/Arrays101/FindEvenDigitsNumbers.cs
public class Solution {
    public int FindNumbers(int[] nums) {
    	int count = 0;
        foreach(int num in nums){
        	count += hasEvenDigits(num);
        }
        return count;
    }

    private int hasEvenDigits(int num){
    	int digits = 0;
    	while(num != 0){
    		digits++;
    		num /= 10;
    	}
		return digits % 2 == 0 ? 1 : 0;
    }
}

File: src/Learn/Arrays101/findEvenDigitsNumbers.cpp
class Solution {
public:
    int findNumbers(vector<int>& nums) {
		int count = 0;
		for(auto num : nums){
			count += hasEvenDigits(num);
		}   
		return count;
    }

    int hasEvenDigits(int num) { 
    	int count = 0;
    	while(num) {
    		num /= 10;
    		count++;
    	}
    	return count % 2 == 0 ? 1 : 0;
    }
};